projects.md

```diff
@@ -18,15 +18,17 @@ Current and past research projects of the Lab.
 </div>
 
 <div class="content list projects">
-  {% for post in site.projects %}
-     {% if post.status == item %}
-     <div class="list-item-projects">
-      <p align="justify" class="list-post-title">
-            <a href="{{ site.baseurl }}{{ post.url }}">{{ post.title }}</a>: {{ post.abstract }}<br>{{ post.grant_number }}<br>
-      </p>
-    </div>
-    {% endif %}
-  {% endfor %}
+	{% for post in site.projects %}
+  	{% if post.status == item %}
+	    <div class="list-item-projects">
+	    	<img src="../images/projects/{{post.logo}}" alt="Project image">
```

Comment:
May I suggest this change? It will pickup the gray figure where there is no project logo.
```suggestion
       {% if post.logo %}
        <img src="{{site.baseurl}}/images/projects/{{post.logo}}" alt="Project image">
        {% else %}
        <img src="{{site.baseurl}}/images/projects/mgray_800x600.png alt="Project image">
        {% endif %}	    	
```